grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlerBuilder.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.grpc;
+
+import static com.google.common.base.Preconditions.checkState;
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Set;
+
+import org.curioswitch.common.protobuf.json.MessageMarshaller;
+
+import com.google.protobuf.Message;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+/**
+ * Constructs a {@link UnframedGrpcErrorHandler} to handle unframed gRPC errors.
+ */
+public final class UnframedGrpcErrorHandlerBuilder {
```

Comment:
```suggestion
@UnstableApi
public final class UnframedGrpcErrorHandlerBuilder {
```

Reply:
When should we use `@Unstable` generally? 
I'm just curious.

Reply:
We add `@UnstableApi` for new public APIs. It is best not to produce any breaking changes. However, it is difficult to make an API stable from the start. `@UnstableApi` is a marker that the API can be changed in a minor update.